x/mongo/driver/operation/hello.go

```diff
@@ -1,3 +1,9 @@
+// Copyright (C) MongoDB, Inc. 2021-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
`hello.go` was missing a license for some reason (the reason being I made the file and forgot to add the license).

Reply:
What's the general procedure deciding when a file should have a license? I'm not super familiar with licensing open source software but it seems pretty important to ensure it's being used the way we expect it to.

Reply:
A great question hahah. I'm _also_ not super familiar with our licensing policy, but this copyright notice should probably appear in every file written by us and available to users of the repo. We're certainly missing the copyright on some files ([here](https://github.com/mongodb/mongo-go-driver/blob/master/mongo/errors_test.go)'s just one example). If it's important for us to have copyright on every file, I can file a ticket for auditing the code base (@kevinAlbs).